test/utils/on-and-off.js

```diff
@@ -64,7 +64,7 @@ function on (tests) {
  */
 exports = module.exports = (tests) => {
   off(tests)
-  on(tests)
+//  on(tests)
```

Comment:
On the pin tests you can select just off by:

```
const runOff = require('../utils/on-and-off').off
describe('pin', () => runOff((thing) => {
```

See bitswap for a only On example: https://github.com/ipfs/js-ipfs/blob/master/test/cli/bitswap.js#L5-L7